src/calculateRentalCost.js

```diff
@@ -5,6 +5,25 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const priceForDay = 40;
```

Comment:
It's better to follow naming conventions and use PRICE_FOR_DAY as a name for this constant